### PR TITLE
Drop py26 support

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
LGTM

File: tox.ini [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26, py27, py34, py35
+envlist = py27, py34, py35, py36, py37
```
[the remark this answers is not in the document]
Comment (MartinBasti) [reply tox.ini]: `¯\_(ツ)_/¯`

I don't know the whole audience which may use this tool.

[the remark this answers is not in the document]
Comment (MartinBasti) [reply tox.ini]: This is available in pypi for masses. IMO we should declare in pypi which versions are supported. Maybe we can add tox to travis

[the remark this answers is not in the document]
Comment (MartinBasti) [reply tox.ini]: pypi classifiers have been added

[the remark this answers is not in the document]
Comment (athos-ribeiro) [reply tox.ini]: LGTM

**Comment by MartinBasti** [comment]
Release notes
> * python 2.6 support has been dropped

**Pull request merged**